.gitignore

```diff
@@ -90,3 +90,4 @@ NuGetExeBuild/
 NuGetRefBuild/
 Packages/
 NuGetTaskBuild/
+build/
```

Comment:
With the explicit excludes, it at least explains how adding a new one would work. Explicitly adding ignored files still work, too. I thus don't feel this is a bigger issue than always having new, untracked files after a build.